elrond-wasm-modules/src/governance/mod.rs

```diff
@@ -74,8 +72,8 @@ pub trait GovernanceModule:
             "Exceeded max actions per proposal"
```

Comment:
Seems like there was also a stored `max_actions_per_proposal`. I'd say remove it entirely and use the constant instead.